src/patterns.md

```diff
@@ -2,8 +2,13 @@
 
 > **<sup>Syntax</sup>**\
 > _Pattern_ :\
+> &nbsp;&nbsp; &nbsp;&nbsp; `|`<sup>?</sup> _PatternNoTopAlt_ `|` _PatternNoTopAlt_
+> &nbsp;&nbsp; | _PatternNoTopAlt_
```

Comment:
I'm not sure I see how this grammar allows more than 2 repetitions.  I would maybe expect something like this:

```suggestion
> &nbsp;&nbsp; &nbsp;&nbsp; `|`<sup>?</sup> _PatternNoTopAlt_ ( `|` _PatternNoTopAlt_ )<sup>\*
```

Reply:
I think you are right. Fixed.